plugins.go

```diff
@@ -48,10 +50,12 @@ var Base = []plugins.Plugin{
 
 	// Generators
 	&ox.Generator{},
+	&liquibase.Generator{},
```

Comment:
@YanDeLeon this one will be optional so no need to add it to the list of plugins.